src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +18,6 @@ public static void main(String[] args) {
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
         executor.shutdown();
```

Comment:
It's good practice to shutdown the ExecutorService after its work is completed to free up system resources. However, you may also want to wait for all tasks to complete before shutting down the executor. Consider using shutdownNow or awaitTermination after allOf to ensure all tasks have completed.

Reply:
done